sysvabi64/sysvabi64.rst

```diff
@@ -0,0 +1,881 @@
+   by systems conforming to it.  Q-o-I is often used to describe the
+   tool-chain-specific means by which a standard requirement is met.
+
+SIMD
+   Single Instruction Multiple Data – A term denoting or qualifying:
+   (a) processing several data items in parallel under the control of one
+   instruction; (b) the Arm v8 SIMD instruction set: (c) the register set
+   shared by (b) and the Armv8 floating point instruction set.
+
+SIMD and floating point
+   The Arm architecture’s SIMD and Floating Point architecture comprising
+   the floating point instruction set, the SIMD instruction set and the
+   register set shared by them.
+
+SVE
+   The Arm architecture's Scalable Vector Extension.
+
+T32
+   The instruction set named Thumb in the Armv7 architecture; T32 uses
+   16-bit and 32-bit instructions.
+
+VG
+   The number of 64-bit “vector granules” in an SVE vector; in other words,
+   the number of bits in an SVE vector register divided by 64.
+
+ILP32
+   SysV-like data model where int, long int and pointer are 32-bit
+
+LP64
+   SysV-like data model where int is 32-bit, but long int and pointer are 64-bit.
+
+LLP64
+   Windows-like data model where int and long int are 32-bit, but long long int and pointer are 64-bit.
+
+This document uses the following terms and abbreviations.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+SysV
+   Unix System V. A variant of the Unix Operating System. Although
+   this specification refers to SysV, many other operating systems,
+   such as Linux or BSD use similar conventions.
+
+Platform
+   A program execution environment such as that defined by an
+   operating system or run- time environment. A platform defines the
+   specific variant of the ABI and may impose additional
+   constraints. Linux is a platform in this sense.
+
+More specific terminology is defined when it is first used.
+
+.. raw:: pdf
+
+   PageBreak
+
+Scope
+=====
+
+Except where otherwise stated the AArch64 System V ABI follows the
+AArch64 base ABI documents in ABIAA_ . The AArch64 System V ABI
+documents the places where divergence exists with respect to the base
+ABI and attempts to act as a unifying document to cover information in
+a variety of places that is of relevance to a System V implementation.
+
+Software Installation
+=====================
+
+This document does not specify how software must be installed on an
+AArch64 system.
+
+Low Level Information
+=====================
+
+TBD
+
+Programming / Coding Examples
+=============================
+
+Architectural Considerations
+----------------------------
+
+The AArch64 architecture does not allow for instructions to encode
+arbitrary 64 bit constants in a single instructions. Immediates or
+constants need to be constructed by a sequence of instructions that
+are defined in the Arm Architecture Reference Manual for AArch64
+ARMARM_. Most instructions accept restricted immediate forms as
```

Comment:
I'd put quotes around ARMARM

Reply:
I've used the full title followed by [`ARMARM`_] as this seems to be how the other 64-bit documents do their references.

Reply:
yea fair enough